### PR TITLE
added all the current staff with root

### PR DESCRIPTION
**Pull request by RonitNath**
@encadyma

### CODE REVIEW

File: modules/ocf_irc/manifests/services.pp [hunk 1]
```diff
@@ -7,7 +7,7 @@
   }
 
   $irc_creds = lookup('irc_creds')
-  $root_nicks = ['waf', 'nattofriends', 'ckuehl', 'jvperrin', 'mattmcal', 'abizer', 'dkessler', 'cooperc', 'jaw']
+  $root_nicks = ['waf', 'nattofriends', 'ckuehl', 'jvperrin', 'mattmcal', 'abizer', 'dkessler', 'cooperc', 'jaw', 'kmo', 'ronitnath', 'njha', 'fydai']
```
Comment (dkess) [inline modules/ocf_irc/manifests/services.pp:10]: This nick isn't registered

**Review by dkess — CHANGES_REQUESTED** [review]

File: modules/ocf_irc/manifests/services.pp (hunk 1 above)
Comment (RonitNath) [reply modules/ocf_irc/manifests/services.pp:10]: oh hm we have to be registered in irc?

**Review by cg505 — APPROVED** [review]
lgtm once dkessler's issue is resolved

File: modules/ocf_irc/manifests/services.pp (hunk 1 above)
Comment (dkess) [reply modules/ocf_irc/manifests/services.pp:10]: yes these all have to be nickserv registered names, they aren't related to OCF accounts

**Review by nikhiljha — APPROVED** [review]

**Pull request merged**